content/docs/pipelines/sdk/python-based-visualizations.md

```diff
@@ -58,9 +58,29 @@ arguments.
 6. Click **Generate Visualization**.
 7. View generated visualization by scrolling down.
 
+## Using TFX visualizations
+
+<video width="960" height="540" controls>
+  <source src="/docs/videos/tfdv_example_with_taxi_pipeline.webm" type="video/webm">
+</video>
+
+1. On Pipelines page, click **[Sample] Unified DSL - Taxi Tip Prediction Model Trainer** to open Pipeline Details page.
+2. On Pipeline Details page, click **Create run**.
```

Comment:
Add "the" before the name of each page, field, and tab, here and in the lines below.

Reply:
Done